tests/unit_tests/test_validators.py

```diff
@@ -417,3 +418,28 @@ def test_to_xml_attrib(min, max, expected_xml):
     xml_validator = validator.to_xml_attrib()
 
     assert xml_validator == expected_xml
+
+
+def test_similar_to_list():
+    # Mock embedding function
+    def embed_function(text: str):
+        """Mock embedding function."""
+        return MOCK_EMBEDDINGS[text]
+
+    # Initialise guard from string
+    guard = Guard.from_string(
+        validators=[SimilarToList()],
+        description="testmeout",
+    )
+
+    output_schema: StringSchema = guard.rail.output_schema
+    data_type: DataType = getattr(output_schema._schema, "string")
+    validators = data_type.format_attr.validators
+    validator: SimilarToList = validators[0]
```

Comment:
You should be able to just instantiate the validator instead of creating a guard

Reply:
Good point! Resolved.